std_msgs/Header header
float32 angle_min_F
float32 angle_max_F
float32 angle_increment_F
float32 time_increment_F
float32 scan_time_F
float32 range_min_F
float32 range_max_F
float32 angle_min_R
float32 angle_max_R
float32 angle_increment_R
float32 time_increment_R
float32 scan_time_R
float32 range_min_R
float32 range_max_R